client/extensions/woocommerce/app/product-categories/form.js

```diff
@@ -34,6 +34,8 @@ class ProductCategoryForm extends Component {
 			id: PropTypes.isRequired,
 		} ),
 		editProductCategory: PropTypes.func.isRequired,
+		onUploadStart: PropTypes.func,
```

Comment:
Should these be `isRequired`? Looks the same for other components where the new props are introduced too.